app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PostsController < ApplicationController
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+  end
+end
```

Comment:
- [ ] Kindly add the index action in your Posts controller.

Reply:
@algerina Hi algerina, I have added an index action in my post controller, thank you very much for reviewing my code.